### PR TITLE
LPS-153032 Change style to fix dropdown width and align collapsble side box

### PR DESCRIPTION
**Pull request by MariMalta**
Story: [[LPS-149014](https://issues.liferay.com/browse/LPS-149014)]

### CODE REVIEW

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by MariMalta** [comment]
ci:stop

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by MariMalta** [comment]
Just started reviewing :)

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ObjectLayoutBox.tsx [hunk 1]
```diff
@@ -99,8 +99,8 @@ const ObjectLayoutBox: React.FC<IObjectLayoutBoxProps> = ({
 							/>
 						</>
 					}
-					displayCollapseIcon
-					displayDragIcon
+					displayCollapseIcon={displayAddButton}
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ObjectLayoutBox.tsx:102]: @cgmonte @MariMalta Please, rename the `displayAddButton` property to something like `regularBlock`

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/HeaderDropdown.tsx [hunk 2]
```diff
@@ -24,14 +24,17 @@ const HeaderDropdown: FC<IHeaderDropdown> = ({
...
 	const flags = useFeatureFlag();
-
 	const [active, setActive] = useState<boolean>(false);
 	const [
 		{
 			isViewOnly,
 			objectLayout: {objectLayoutTabs},
 		},
 	] = useContext(LayoutContext);
+	const handleOnClick = (handler: Function) => {
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/HeaderDropdown.tsx:34]: @cgmonte @MariMalta Please, add a blank line here. The `setActive` function is being defined in this same block as its usage.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by MariMalta** [comment]
ci:forward

**Pull request closed without merging**